Wrappers/Python/test/test_ISTA_algorithm.py

```diff
@@ -101,6 +102,49 @@ def test_update(self):
         res1 = ista.objective[-1]
         res2 = self.f(x) + self.g(x)
         self.assertTrue( res1==res2) 
+        
+    def test_update_g_none(self):
+
+        # ista run 10 iteration
+        tmp_initial = self.ig.allocate()
+        ista = ISTA(initial = tmp_initial, f = self.f, g = None,  max_iteration=1)  
+        ista.run()
+
+        x = tmp_initial.copy()
+        x_old = tmp_initial.copy()
+
+        for _ in range(1):         
+            x = ista.g.proximal(x_old - (0.99*2/ista.f.L) * ista.f.gradient(x_old), (1./ista.f.L))
+            x_old.fill(x)
```

Comment:
why in range(1)? Can this just be written as one iteration by hand without the loop?